server/src/main/java/io/seata/server/storage/db/session/DataBaseSessionManager.java

```diff
@@ -79,7 +77,7 @@ public DataBaseSessionManager(String name) {
 
     @Override
     public void init() {
-        transactionStoreManager = EnhancedServiceLoader.load(TransactionStoreManager.class, StoreMode.DB.name());
+        transactionStoreManager = new DataBaseTransactionStoreManager();
```

Comment:
Will be initialized 4 times by SessionHolder, it should be static.

Reply:
done